lavamoat/browserify/main/policy.json

```diff
@@ -1729,6 +1785,38 @@
         "bitcoin-address-validation": true
       }
     },
+    "@metamask/assets-controllers>@metamask/keyring-api": {
```

Comment:
This (and other similar entries) is also expected, since we're only bumping the `accounts-controller` dep for now, resulting in a different major versions for the `keyring-api` for those packages.